src/apiCalls.js

```diff
@@ -1,7 +1,7 @@
 export const fetchApiData = (type) => {
   return fetch(`http://localhost:3001/api/v1/${type}`)
     .then(response => response.json())
```

Comment:
So I assume this function is where you're having trouble dealing with displaying a get error? If so, I believe you need to do the same thing here that you've already done with post and delete....remove .then and .catch from here and put them where you call fetchApiData in scripts. It is my understanding then you might still need to check status as a part of GET. I think it will be easier to invoke your show error message function if your .then and catch are in scripts

Reply:
I added a commit after our call about this one. Thanks for helping me figure it out!!!